hw02/hw02.R

```diff
@@ -0,0 +1,64 @@
+library(dplyr)
+library(gapminder)
+gapminder$country
+#1.1
+data1.1 <- gapminder %>% 
+  filter(country == "Afghanistan"|country == "Albania"|country == "Angola", year < 1980 & year > 1969)  
+#1.2
+data1.1 %>% 
+  select(country, gdpPercap)
+#1.3
+lifeExpchange1 <- gapminder %>% 
+  arrange(year) %>% 
+  mutate(lifeExpchange = lifeExp - lag(lifeExp))
+lifeExpchange1 %>% 
+  filter(lifeExpchange<0)
+#1.4
+gapminder %>%
+  group_by(country) %>%
+  summarize(maxGdpPercap = max(gdpPercap))
+#1.5
+gapminder %>%
+  filter(country=="Canada") %>% 
+  ggplot() +
+  aes(x = gdpPercap,
+      y = lifeExp) +
+  geom_point() +
+  scale_x_log10()
+
+
+#2.1
+palmerpenguins::penguins
+palmerpenguins::penguins %>% 
+  summarize(mu1=mean(bill_length_mm, na.rm = T), sigma1=sd(bill_length_mm, na.rm = T), median1=median(bill_length_mm, na.rm = T),
+            mu2=mean(flipper_length_mm, na.rm = T), sigma2=sd(flipper_length_mm, na.rm = T), median=median(flipper_length_mm, na.rm = T))
+#####or class notes
+#palmerpenguins::penguins %>% 
+#  summarize(across(c(bill_length_mm, flipper_length_mm), list(mean= ~ mean(.x, na.rm=T), sd=sd)))
+
+library(ggplot2)
+ggplot(palmerpenguins::penguins) +
```

Comment:
Great job so far! Don't forget to use the penguins' raw data for the plot instead of the regular data.